en/02_Developer_Guides/02_Controllers/02_Routing.md

```diff
@@ -303,6 +303,53 @@ class BreadAPIController extends Controller
 In Silverstripe CMS versions prior to 4.6, an empty key (`''`) must be used in place of the `'/'` key. When specifying an HTTP method, the empty string must be separated from the method (e.g. `'GET '`). The empty key and slash key are also equivalent in Director rules.
 [/alert]
 
+## Nested RequestHandlers
```

Comment:
```suggestion
## Nested `RequestHandler` instances
```
RequestHandler is a class name - it must be in backticks.
Alternatively, you could do this:

```suggestion
## Nested request handlers
```
In that case we're not referring to the class name, but the general concept of a request handler - so it has to be a sentence style header.